docs/data-collection/scanning-notes.md

```diff
@@ -36,6 +36,11 @@
 
     ![adjustFOV.jpg](../assets/images/adjustFOV.jpg)
 
+### Repeat scan
+- [ ] In case, you have to interrupt a sequence because a problem occured (e.g your participant fell asleep, the physiological recordings did not start etc.) or you have to repeat a sequence because the image was of bad quality, right click on the sequence that needs to be restarted and click on `Repeat` to restart the scan **without changing anything in its name!**
```

Comment:
```suggestion
- [ ] If you have to interrupt a sequence because a problem occurred (e.g., the participant fell asleep, the stimuli were not adequately started, etc.), or you have to repeat a sequence because the image was of low quality, right click on the sequence that needs to be restarted and click on *Repeat* to restart the scan **without changing anything in its name!**
```